assets/fonts/OpenSans/font-face.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
```

Comment:
It's a light font, PT Sans have no light version, so I found another one. Also this font will be used on home page.